compose/config/errors.py

```diff
@@ -1,3 +1,7 @@
+class ConfigError(ValueError):
+    pass
```

Comment:
A TODO about this being legacy and us needing to consolidate to use the `ConfigurationError` would be good here I think